examples/basic-webpack/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "basic-webpack-example",
+  "version": "1.0.0",
+  "description": "React example with webpack",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack",
+    "dev-server": "webpack-dev-server"
```

Comment:
Can we make these match up with what we have in the commonjs example? `build` would just run `webpack` and `start` would run the devserver.